crawler.py

```diff
@@ -93,46 +78,44 @@ def download_and_format(id: str) -> Study:
 
     #foramtting 
     title = data.find("official_title").get_text()
-    description = "" #TODO #summary?
-        
+    description = data.find("brief_summary").get_text()
+            
     date_str = data.find("last_update_posted").get_text()
     last_updated = date.strptime(date_str, '%B %d, %Y')
-        
+            
     study_type = data.find("study_type").get_text()
-    condition = [] #TODO 
-    sponsor = data.find("lead_sponsor").find("agency").get_text() ##lead sponsor and collborators
-    recruitment_status = data.find("overall_status").get_text()
-        
-    min_age = data.find("minimum_age").get_text()
-    max_age = data.find("maximum_age").get_text()
-    unit = min_age[min_age.index(" ")+1:]
-    age = (int("".join(filter(str.isdigit, min_age))), 
-            int("".join(filter(str.isdigit, max_age))), unit)
+
+    conditions = []
+    for condition in data.findAll("condition"):
+        conditions.append(condition)
         
+    sponsor = data.find("lead_sponsor").find("agency").get_text() ##lead sponsor only (no collborators)
+    recruitment_status = data.find("overall_status").get_text()
+            
+    age = {"min":data.find("minimum_age").get_text(), "max":data.find("maximum_age").get_text()}
+            
     sex = data.find("gender").get_text()
-    control = "" #TODO #??
-    additional_criteria = "" #TODO #inclusion/exclusion
+    control = data.find("healthy_volunteers").get_text()
+    additional_criteria = data.find("criteria").get_text()
 
     locations = []
-    for loc in data.findAll("location"):
-        name = loc.find("name").get_text()
-        country = loc.find("country").get_text()
-        city = loc.find("city").get_text()
-        zipcode = loc.find("zip").get_text()
-        status = loc.find("status").get_text()
-        locations.append({"name": name, "country": country, "city": city, "zipcode": zipcode, "status": status})
-
-    contact = {}
-    c = data.find("overall_contact")
-    if c is not None:
-        contact["name"] = c.find("last_name").get_text()
-        contact["phone"] = c.find("phone").get_text()
-        contact["email"] = c.find("email").get_text()
+    if data.findAll("location") is not None:
+        for loc in data.findAll("location"):
+            localLocation = loc.find("name").get_text()
+            nationalLocation = loc.find("address").get_text().strip("\n").replace("\n", ", ")
+            locations.append({"locationLocation": localLocation, "nationalLocation": nationalLocation, "status": status})
```

Comment:
The status part also caused an error, do we need it? maybe we can do something like this
```suggestion
            locations.append({"locationLocation": localLocation, "nationalLocation": nationalLocation})
```